src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -50,24 +50,6 @@ export class ArtworkApp extends React.Component<Props> {
     this.trackLotView()
   }
 
-  componentDidUpdate(prevProps) {
```

Comment:
We don't need this as on route change a full page re-rerender occurs -- including componentDidMount.

Reply:
This feels like it conflicts with the behavior we observed in the first AB test. Namely, when navigating to an artwork from another artwork (or within any of the same app), the app wouldn't get re-mounted. Instead, it's a params/props update.

Or, do you mean that with our new tracking middleware (responding to router events), we can simply drop this React lifecycle hook entirely? Cause that does make sense to me, but I feel like your explanation is a bit different than my understanding.

Reply:
(I would also have a follow-up, that if the component doesn't re-mount but our tracking middleware will take care of the page view, maybe this should be reinstated, but _without_ the pageview (and with the custom product and lot view events)).

Reply:
Adding a log to `componentDidMount` and tracking and clicking between artworks triggers a proper event from componentDidMount each time -- it rerenders the whole page. Nothing strange?